helloworld/tests/cpp_wall_clock/stall.cpp

```diff
@@ -0,0 +1,16 @@
+#include <time.h>
+#include <stdio.h>
+#include <stdint.h>
+void gettime(uint64_t *w) {
```

Comment:
Get some spacing here. I thought I only have to point out one instance of the problem, but your feeble mind clearly does not understand something so trivial.